includes/definitions/avocent-mp.yaml

```diff
@@ -0,0 +1,40 @@
+os: avocent-mp
+text: Vertiv Avocent MergePoint Unity
+type: appliance
+icon: vertiv
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.10418.18.1
+discovery_modules:
+    cisco-vrf-lite: false
+    bgp-peers: false
+    wireless: false
+    mpls: false
+    ports-stack: false
+    entity-physical: false
+    sensors: false
+    storage: false
+    hr-device: false
+    discovery-protocols: false
+    vlans: false
+    ucd-diskio: false
+    stp: false
+    ntp: false
+    fdb-table: false
+poller_modules:
+    bgp-peers: false
+    entity-physical: false
+    hr-mib: false
+    ipSystemStats: false
+    ipmi: false
+    mpls: false
+    ntp: false
+    ospf: false
+    storage: false
+    stp: false
+    ucd-mib: false
+    ucd-diskio: false
+    wireless: false
```

Comment:
you should remove these, we only force disable them if the device have troubles with them